### DISCUSSION TITLE
lab3在线评测测例performance_test2021-public /conv0.sysu.c

### DISCUSSION THREAD

**Discussion by 91Mrcui** [arcsysu/SYsU-lang#112] [Q&A]
这个测例的函数memmove解析不了，要换个名字例如换成mem_move，请问是什么问题导致的，该如何解决？

**Comment by Zhou7D663** [comment]
可以看看clang生成的json格式的抽象语法树，我记得memmove是内置函数，会有两个FunctionDecl 
可以看看这个discussion #108 
performance_test2021-public /conv0.sysu.c的内容可以查看这里[conv0.sysu.c](https://github.com/arcsysu/SYsU-lang-tester-perfermance/blob/9dcd4d220cab394cb907e0cf962cd9309444e5ef/performance_test2021-public/conv0.sysu.c)

**Reply by Zhou7D663** [reply]
后面的测例中sqrt函数也是同样的问题
